tests/test_routes/test_watch_routes.py

```
@@ -0,0 +1,192 @@
+    """POST /api/watch/start のテスト"""
+
+    def test_観戦モードを正常に開始(self, csrf_client):
+        """観戦モードの正常な開始"""
+        with patch("app.initialize_llm") as mock_llm:
+            mock_llm_instance = MagicMock()
+            mock_llm.return_value = mock_llm_instance
+
+            with patch(
+                "services.watch_service.WatchService.generate_initial_message"
+            ) as mock_gen:
+                mock_gen.return_value = "こんにちは！今日はいい天気ですね。"
```

Ruff (0.14.6)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)